### ISSUE TITLE
Another fake BMC Ethernet device that needs to be ignroed by the resource jobs

### ISSUE THREAD

**Issue by bladernr** [canonical/checkbox#274]
### Bug Description

Yet another fake ethernet device from a BMC that is being incorrectly identified as a NIC for testing.
Category: NETWORK
Interface: enx2a508835222b
Product: Virtual Ethernet.
Vendor: American Megatrends, Inc.
Driver: cdc_ether
Driver Version: 
Path: /devices/pci0000:00/0000:00:14.0/usb1/1-11/1-11.3/1-11.3:2.0
Id: [046b:ffb0]
Mac: 2a:50:88:35:22:2b
Carrier Status: Connected
Ipv4: ***NOT CONFIGURED***
Ipv6: fe80::2850:88ff:fe35:222b/64
Speed: -1
Supported Modes: 
	Not reported
Advertised Modes: 
	Not reported
Partner Modes: 

### To Reproduce

Huh/ 

### Environment

BMC

### Relevant log output

```shell
- Checkbox session(s) (located in `/var/tmp/checkbox-ng/sessions/`, you usually want to select the most recent one)

- logs from the impacted components (e.g. `lsblk` if this is related to an issue when testing a disk...); a safe option is to install and run `sosreport` to gather as much log as possible.
```
